.github/workflows/ci.yaml

```diff
@@ -64,6 +65,7 @@ jobs:
     - uses: actions/setup-go@v3
       with:
         go-version: v1.18
+        cache: true
     - run: make build
```

Comment:
Can we skip this `make build` step in each job? Both `test-e2e` tagets rely on `build-all`, and if they're not using the output of `make build`, it seems like a waste.

Reply:
We need to have it here to have the binaries available for the image builds.